test/runtests.jl

```diff
@@ -4,6 +4,7 @@ DocMeta.setdocmeta!(Pickle, :DocTestSetup, :(using Pickle); recursive=true)
 
 include("./pyscript.jl")
 include("./torch/thscript.jl")
+include("sparse.jl")
```

Comment:
shoudn't add it here.

rename `test/sparse.jl` as `test/test_sparse.jl` and add a `"sparse",` entry at the test list https://github.com/chengchingwen/Pickle.jl/blob/456335717808b41463f0063ce36208cb78187fda/test/runtests.jl#L24-L28